deployment/src/main/resources/templates/api.qute

```diff
@@ -4,6 +4,10 @@ package {package};
 import {imp.import};
 {/for}
 
+{#if mutiny}
+import io.smallrye.mutiny.Uni;
+{/if}
```

Comment:
```suggestion
```

Rather than importing, the current approach we've been taking is to use the full qualified class name. By using FQCNs we avoid conflicts with user types.